### PR TITLE
De subtitles anita malavasi 02

### PR DESCRIPTION
**Pull request by kaholli**

### CODE REVIEW

File: locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml [hunk 1]
```diff
@@ -261,17 +249,17 @@ de:
...
           from: 00:04:56,13
           to: 00:05:05,08
-          text: I had made friends with the young men who came to our bar.
+          text: Ich hatte Freunschaft geschlossen mit den jungen Männern, die in unsere Bar kamen.
         sequence-50:
           s: 50
           from: 00:05:05,08
           to: 00:05:10,29
-          text: When they left for war I became their communication point.
+          text: Als sie in den Krieg gingen, wurde ich ihr Kommunikationspunkt.
```
Comment (kaholli) [inline locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml]: Ich finde das Wort "Kommunikationspunkt" irgendwie unschön, aber mir ist nichts besseres eingefallen. Bin offen für Ideen.

File: locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml [hunk 2]
```diff
@@ -150,108 +144,102 @@ de:
...
           to: 00:03:50,03
-          text: Reggiane factory technicians of the water and gas companies.
+          text: Reggio Fabriktechniker der Wasser- und Gasfirmen.
         sequence-40:
           s: 40
           from: 00:03:50,03
           to: 00:03:57,12
-          text: They were a very bright, mature and well-prepared group of working
-            class people.
+          text: Es war sehr intelligente, reife und gut vorbereitete Gruppe von Leuten der Arbeiterklasse.
```
Comment (pixelpogo) [inline locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml]: I'm not really sure about this sentence... Maybe someone with Italian knowledge can help out here?
@busfaktor2 @ERA-aishah @eabalos 
"c’è un tipo di classe operaia molto consapevole, molto matura, molto preparata...", see http://www.resistance-archive.org/downloads/transcripts/anita-malavasi-it.pdf

File: locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml (hunk 1 above)
Comment (pixelpogo) [inline locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml]: Google Translate says `Bezugspunkt`.
What is the opinion of the Italian Experts(@busfaktor2 @ERA-aishah @eabalos @GContaldi @sgrippo @giulia1983)? Again the Italian transcript is available on http://www.resistance-archive.org/downloads/transcripts/anita-malavasi-it.pdf

Comment (kaholli) [inline locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml]: I would go for "Bezugsperson" now?

**Comment by pixelpogo** [comment]
Thanks for your contribution, @kaholli 
Will merge the PR now.

**Pull request merged**